ls1intum/Artemis
# [Test] More exam management tests

### Motivation and Context


We want more Cypress tests for the exam functionality.

### Description

This PR adds tests for exercise group creation, adding a programming exercise to it, registering course students to the exam and generating individual student exams. It also provides pageobjects which can be reused in future tests. I also added a subfolder for all exam related spec files to keep the integration test folder structured.

### Steps for Testing


1. Configure cypress for your test server of choice according [to this page](https://confluence.ase.in.tum.de/display/ArTEMiS/Cypress+Tests)
2. Run `yarn cypress:run --spec src/test/cypress/integration/exam/ExamManagement.spec.ts`
3. Make sure that all tests passed
